hs_access_control/models/community.py

```diff
@@ -5,6 +5,7 @@
 
 from hs_core.models import BaseResource
 from theme.utils import get_upload_path_community
+from sorl.thumbnail import ImageField as SolrImageField
```

Comment:
I was confused by SolrImageField... should it be SorlImageField?

Reply:
Good call thanks. I changed it to `TumbnailImageField`